# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/cocoa/moz.build
# Commit: 8287c3d0c106
# Full Hash: 8287c3d0c1064096ee7c162060e6615d72722ee0
# Author: Christoph Walcher <christoph-wa@gmx.de>
# Date: 2019-06-20 09:46:31
# Regressor Bug: 909760
# File Overlap Count: 1
# Description:
#   Bug 909760 - Show download progress in the MacOS Finder r=spohl,mak
#   
#   Show download progress in the MacOS Finder
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30688
# ==============================================================================

diff -r 5b9a3de04646 -r 8287c3d0c106 widget/cocoa/moz.build
--- a/widget/cocoa/moz.build	Wed Jun 19 18:39:02 2019 +0000
+++ b/widget/cocoa/moz.build	Wed Jun 19 18:48:49 2019 +0000
@@ -39,6 +39,7 @@
     'nsLookAndFeel.mm',
     'nsMacCursor.mm',
     'nsMacDockSupport.mm',
+    'nsMacFinderProgress.mm',
     'nsMacSharingService.mm',
     'nsMacWebAppUtils.mm',
     'nsMenuBarX.mm',